app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -171,6 +172,8 @@ fun CommentBodyPreview() {
 
 fun LazyListScope.commentNodeItem(
     node: CommentNode,
+    admins: List<PersonView>,
+    moderators: List<CommunityModeratorView>?,
```

Comment:
I don't like the fact that we have to inject these lists for every single comment, and I'm sure its a performance hit. But this canMod logic unfortunately lives in the front end, and requires both.

Reply:
These need to be `ImmutableList`, all of them. I saw that you used the correct immutablelist at the source but not in the parameters.

*Difference btw ImmutableList vs List, only one of them is the compose compiler sure that it will be immutable*
![image](https://github.com/dessalines/jerboa/assets/67873169/d237c0fd-ada5-499d-b8db-0d0c0d1aafdc)


The performance shouldn't be impacted much. I see that it only runs when opening the options menu.

Reply:
We can avoid these prop drilling. But you won't like the solution either. We can do DI and inject them directly into MoreOptions menu

Reply:
I can technically simulate this already in the LemmyAPI. If you prefer this?